weakmw.tex

```diff
@@ -201,23 +250,32 @@ \section{Proof of the Weak Mordell-Weil Theorem}
 	injective uses \emph{formal groups}, whose development is outside the
 	scope of this course.\footnote{For an introduction to
 	formal groups of elliptic curves see \cite[Ch. IV]{silverman:aec}.}
-	Next, as above, $\sigma(Q)-Q \in E(K)[n]$ for
+	Next fix any $Q$ as in the statement of the theorem.
+	Above we saw $\sigma(Q)-Q \in E[n]$ for
 	all $\sigma \in \Gal(\Kbar/K)$.  Let $I_\p\subset \Gal(L/K)$ be the
-	inertia group at $\p$.  Then by definition of interia group, $I_\p$
+	inertia group at $\p$. By definition of interia group, $I_\p$
```

Comment:
"By definition of interia group" --> "Be definition of the inertia group" (TWO CHANGES!!!!)   Interia!